src/data/string/ustring.h

```diff
@@ -105,14 +109,19 @@ uchar chars_to_uchar_impl(InputIterator1& in, InputIterator2 end)
       break;
     }
 
-  if (nbytes == 0)
-    throw std::invalid_argument("Invalid UTF-8");
+  // There is no problem by using (nbytes == 2).
+  // But nbytes(>= 2) is used later, so (nbytes < 2) is used here for readability.
+  if (nbytes < 2) {
+    throw std::invalid_argument("Invalid UTF-8: UTF-8 byte sequences have a 5-byte or 6-byte sequence");
```

Comment:
When continuation byte comes first byte of character, it is checked at previous condition("if (c < 0xC0 || c > 0xFD)") and reported as error.